src/configuration/index.ts

```diff
@@ -11,7 +11,23 @@ configurationRegistry.registerDefaultConfigurations([{
     'editor.fontSize': 12,
     'editor.maxTokenizationLineLength': 1000,
     'editor.quickSuggestions': false,
-    'files.eol': '\n'
+    'files.eol': '\n',
+    'editor.semanticTokenColorCustomizations': {
+      rules: {
+        '*.static': {
+          fontStyle: 'italic'
+        },
+        '*.final.static': {
+          fontStyle: 'italic bold'
+        },
+        '*.readonly': {
+          fontStyle: 'bold'
+        },
+        '*.deprecated': {
+          fontStyle: 'strikethrough'
+        }
+      }
+    }
```

Comment:
will we be able to remove those from the themes now that it's here ?

Reply:
It's not new, it was just moved, I think we already were able to not have it in the themes